tests/functional/graph-equivalence/00-oneline.t

```diff
@@ -25,24 +25,21 @@ set_test_number 5
 install_suite "${TEST_NAME_BASE}" test1
 #-------------------------------------------------------------------------------
 TEST_NAME="${TEST_NAME_BASE}-validate"
-run_ok "${TEST_NAME}" cylc validate "${SUITE_NAME}"
+run_ok "${TEST_NAME}" cylc validate \
+    --set=TEST_OUTPUT_PATH="${PWD}" "${SUITE_NAME}"
 #-------------------------------------------------------------------------------
 TEST_NAME="${TEST_NAME_BASE}-run"
-suite_run_ok "${TEST_NAME}" \
-    cylc run --reference-test --debug --no-detach "${SUITE_NAME}"
+suite_run_ok "${TEST_NAME}" cylc run --reference-test --debug --no-detach \
+    --set=TEST_OUTPUT_PATH="${PWD}" "${SUITE_NAME}"
```

Comment:
Ha-ha, I was wondering why the changes in this test and the associated `test1/flow.cylc`, then thought why in hell is this test passing on master... but I see you explained it:
> The other one; cylc show --list-prerequisites only shows prerequisites for live tasks, which may be rectified post n-window

Reply:
(and same for the other graph-equivalence tests)